sdk/search/azure-search-documents/pom.xml

```diff
@@ -29,7 +29,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.4.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.5.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
```

Comment:
Shouldn't the version tag also be updated when we switch to unreleased dependency?

Reply:
This is for test purpose. I am going to revert back to the released version as we do not depend on any new Version functionality for now.